src/screens/ExchangeModal.tsx

```diff
@@ -995,10 +1101,11 @@ export default function ExchangeModal({
                   onPressSelectOutputCurrency={() =>
                     navigateToSelectOutputCurrency(chainId)
                   }
-                  {...(currentNetwork === Network.arbitrum &&
-                    !!outputCurrency && {
-                      onTapWhileDisabled: handleTapWhileDisabled,
-                    })}
+                  onTapWhileDisabled={
```

Comment:
this is gonna break the behavior on output token input related to https://github.com/rainbow-me/rainbow/pull/4175#issuecomment-1260001766, lint screams about this but if we don't {... } is gonna be broken 😢

Reply:
or, maybe there's a better solution under the hood, but no need to be addressed on this PR imo

Reply:
Happy mto put it back in I have a feeling this was a bad merge on my part